app/src/hooks/useGraphMarketsFromQuestion.tsx

```diff
@@ -50,13 +51,10 @@ export const useGraphMarketsFromQuestion = (questionId: string): Result => {
     if (!questionId) setMarkets([])
   }, [questionId])
 
-  if (data && data.question && data.question.indexedFixedProductMarketMakers.length > 0 && markets.length === 0) {
+  if (data && data.question && !isObjectEqual(markets, data.question.indexedFixedProductMarketMakers)) {
     setMarkets(data.question.indexedFixedProductMarketMakers)
   } else if (data && data.question && !data.question.indexedFixedProductMarketMakers.length) {
-    return {
-      markets: [],
-      status: Status.Error,
-    }
+    setMarkets([])
```

Comment:
Shouldn't we still return an error here?

Reply:
@KadenZipfel , this section doesn't mean error.
It means "no markets" is returned.